source/nanoFramework.Json/nanoFramework.Json.nfproj

```diff
@@ -72,11 +77,11 @@
       <ProjectConfigurationsDeclaredAsItems />
     </ProjectCapabilities>
   </ProjectExtensions>
-  <Import Project="..\packages\Nerdbank.GitVersioning.3.1.91\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.1.91\build\Nerdbank.GitVersioning.targets')" />
+  <Import Project="..\packages\Nerdbank.GitVersioning.3.2.5-beta\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.2.5-beta\build\Nerdbank.GitVersioning.targets')" />
```

Comment:
Please dont update this nuget, we need the stable one for the release pipeline

Reply:
Interesting.  This created some trouble for me.  I'll revert back to 3.1.91